pkg/reconciler/managed/reconciler.go

```diff
@@ -42,8 +42,7 @@ const (
 	reconcileGracePeriod = 30 * time.Second
 	reconcileTimeout     = 1 * time.Minute
 
-	defaultManagedShortWait = 30 * time.Second
-	defaultManagedLongWait  = 1 * time.Minute
+	defaultManagedLongWait = 1 * time.Minute
```

Comment:
Nit: Maybe long doesn't make sense in context anymore, since we don't have a short wait? I tend to think of this as the "speculative" or "happy path" poll interval these days.

Reply:
@negz I was considering that as well, but didn't know if we want to the `WithLongWait()` on update. I think most providers don't actually use that option (i.e. they default to the 1 minute), and it might also be good for the break to be exposed to give a stronger indication that something has changed behind the scenes.

Reply:
We'd be making a fairly equivalent breaking change in removing `WithShortWait()`, right? We could also have WithLongWait become an alias for `WithPollInterval` or whatever we called this now.